server.js

```diff
@@ -29,6 +29,32 @@ app.post('/project',function(req , res){
 		res.send(data)
 	})
 })
+
+//Routes for Tasks :)
+app.get('/tasks', function(req, res)
+{
+	db.Task.find({}, function(err, data)
+	{
+		if(err)
+		{
+			res.send(err);
+		}
+		res.status(200).send(data);
+	});
+});
+
+app.post('/tasks', function(req, res)
+{
+	db.addTask(req.body, function(err, data)
```

Comment:
Once again, make sure opening bracket ('{') is on the same line as the function keyword.